<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SIF</class>
 <widget class="QWidget" name="SIF">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>340</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Comic Sans MS</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Логинимся</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Залогиниться!</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;p&gt;Привет! Меня зовут &lt;strong&gt;Ай Амано&lt;/strong&gt;, я веселая девочка - искусственный интеллект! Да-да, мое имя как раз от аббревиатуры английского &lt;i&gt;Artificial Intelligence&lt;/i&gt; - &quot;искусственный интеллект&quot;!&lt;/p&gt;
&lt;p&gt;А ты кто? Давай познакомимся! Для этого залогинься в меня.&lt;/p&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_login">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>271</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Логин</string>
   </property>
  </widget>
  <widget class="QLabel" name="error">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_passwd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>271</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="wordgame.qrc"/>
 </resources>
 <connections/>
</ui>
